CHANGELOG.md

```diff
@@ -1,6 +1,69 @@
 # Changelog
 
+## [0.4.0] - 2022-01-14
```

Comment:
need update links in line 126 to make those links work

checkout https://keepachangelog.com/en/1.0.0/